### ISSUE TITLE
Doesn't track renew rank 10

### ISSUE THREAD

**Issue by yarvaxea** [laytya/Chronometer-vanilla#17]
I can't get Chronometer to track the max rank renew from AQ20. It will track rank 9 perfectly but not rank 10.

**Comment by laytya** [comment]
its looks different in buffs? can u make screens?

**Comment by yarvaxea** [comment]
The tracker doesn't appear at all for rank 10.
![rank 9 renew](https://user-images.githubusercontent.com/26580922/59434958-dfea8e00-8dec-11e9-8b76-92d424b9dc3b.PNG)
As you can see it tracks rank 9 perfectly well.
![max rank renew](https://user-images.githubusercontent.com/26580922/59434965-e2e57e80-8dec-11e9-9e32-d8475421f1ac.PNG)
But with rank 10 it doesn't appear. 

Has it something to do with rank 10 being from the AQ20 books?

**Comment by laytya** [comment]
download fixed version

**Comment by yarvaxea** [comment]
Okay this is really strange. If I click the spell from the spellbook it tracks rank 10 renew as well, but if I use my mouseover macro it doesn't track it... It does track mouseover rank 4 though. I discovered what the problem was. In my mouseover macros the rank 10 was called:
`/lunamo renew(Rank 10)`
and the rank 4 was called:
`/lunamo Renew(Rank 4)`
So the problem must be that the spell has to be named with capital first letter. When I changed the spelling it works. Really odd.
